### PR TITLE
feat(context-selector): add period select

### PR DESCRIPTION
**Pull request by HendrikThePendric**
**TL;DR;**
ignore `src/top-bar/period-select/fixed-periods.js` and `src/top-bar/period-select/fixed-periods.test.js` during your review.

**Why?**
The file `src/top-bar/period-select/fixed-periods.js` was taken from the analytics package and moved here as a temporary measure. In the long run this logic should be extracted into its own package. I've renamed a few things but have kept the vast majority of the internal logic the same. As such, I would argue that reviewing this code is out-of-scope for this PR, and actually out-of-scope for the approval-app. Instead this should be reviewed before we introduce that package for period manipulations....

### CODE REVIEW

**Review by Mohammer5 — COMMENTED** [review]
The start year (`new Date(0)`), I assume that's just a placeholder until a proper value can be retrieved?

---

The PeriodMenu renders `null`, I guess this is also just a placeholder? If not, should we display a message of some kind?

**Review by Mohammer5 — APPROVED** [review]
Otherwise, LGTM
